docs/tutorials.rst

```diff
@@ -19,10 +19,21 @@ STAC Metadata Visualization
 This tutorial gives an introduction to using Holeviews and hvplot to visualize
 STAC metadata and Item geometries on a map.
 
-CQL Filtering
+CQL2 Filtering
 ---------------------------
 
 - :tutorial:`GitHub version <cql2-filter.ipynb>`
+- :ref:`Docs version </tutorials/cql2-filter.ipynb>`
```

Comment:
@gadomski I didn't add it to the Metadata Visualization one b/c it needs additional libraries, and PROJ 8 is a pain to install on Ubuntu